encryption/README.md

```diff
@@ -0,0 +1,26 @@
+# Encryption
+
+Create a custom data converter that encrypts data with AES. See [encryption docs](https://docs.temporal.io/docs/typescript/data-converters#encryption).
+
+## Code
+
+- `EncryptionCodec`: [encryption-codec.ts](https://github.com/temporalio/samples-typescript/blob/main/encryption/src/encryption-codec.ts)
```

Comment:
Can use relative paths here (granted unsure about npm, but I figure we aren't putting each sample there as its own project anyways)